ush/templates/FV3LAM_wflow.xml

```diff
@@ -1346,29 +1346,8 @@ MODULES_RUN_TASK_FP script.
 
     <dependency>
        <and>
-         <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
-         <taskdep task="&PREP_CYC_TN;_#type#{{ uscore_ensmem_name }}"/>
-         {%- if use_rrfse_ens %}
-         {%- if do_retro %}
-         <or>
-           <and>
-             <or>
-             {%- for h in cycl_hrs_hyb_fv3lam_ens %}
-               <streq><left>{{ h }}</left><right><cyclestr>@H</cyclestr></right></streq>
-             {%- endfor %}
-             </or>
-             {% for h in range(1, num_ens_members+1) %}
-             <datadep age="00:00:01:00"><cyclestr offset="-{{ da_cycle_interval_hrs }}:00:00">&RRFSE_FG_ROOT;/@Y@m@d@H/mem{{ "%04d" % h  }}/fcst_fv3lam/RESTART/</cyclestr><cyclestr>@Y@m@d.@H0000.coupler.res</cyclestr></datadep>
-             {%- endfor %}
-           </and>
-           <and>
-              {%- for h in cycl_hrs_hyb_fv3lam_ens %}
-                <strneq><left>{{ h }}</left><right><cyclestr>@H</cyclestr></right></strneq>
-              {%- endfor %}
-           </and>
-         </or>
-         {%- endif %}
-         {%- endif %}
+         <timedep><cyclestr offset="&START_TIME_PROCPM;">@Y@m@d@H@M00</cyclestr></timedep>
+         <taskdep task="&PROCESS_PM_TN;_spinup"/>
```

Comment:
should have dependence to 
`<taskdep task="&PREP_CYC_TN;_#type#{{ uscore_ensmem_name }}"/>`